### ISSUE TITLE
branch: graphTopLayer

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/energy-skate-park#38]
To get labels on the graph to look like https://github.com/phetsims/energy-skate-park/issues/25#issuecomment-432714812, but this was problematic because then the skater is underneath the labels, but on top of the bars in the graph and it looks strange.

Going to try something else but I didn't want to lose these changes, so storing in a branch.

**Comment by jessegreenberg** [comment]
After #66 this is no longer relevant and can be pruned.

**Comment by jessegreenberg** [comment]
Deleted.

**Comment by jessegreenberg** [comment]
This was re-deleted after branches were restored across the project.
